### PR TITLE
storybook and readme

### PR DESCRIPTION
**Pull request by fabianovangool**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## Stack
+
+The technologies used in this project are:
+
+- Prettier: for code formatting
+- ESLint: for code linting
+- Typescript: for more reliable code
+- Vercel: for deployment of the app on GitHub
+- Jest: for running unit and end-to-end tests
+- Storybook:
```
Comment (nathaliebru) [inline README.md:40]: For consistency add information after the colon.

**Review by nathaliebru — COMMENTED** [review]
Looks good!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#To install the necessary dependencies from the package.json:
+npm i
+```
+
+## Stack
+
+The technologies used in this project are:
+
+- Prettier: for code formatting
+- ESLint: for code linting
```
[the remark this answers is not in the document]
Comment (fabianovangool) [reply README.md:36]: Not sure actually!

**Review by nathaliebru — DISMISSED** [review]
Very niiiice!

**Review by nathaliebru — APPROVED** [review]

**Review by hans-acc — APPROVED** [review]
shimmer and shine 🌟

**Pull request merged**